parts/k8s/kubernetesmastercustomscript.sh

```diff
@@ -453,28 +644,39 @@ users:
     set -x
 }
 
+ensureRunCommandCompleted
```

Comment:
I also assume re-jiggering the operational queue by moving `ensureRunCommandCompleted` to come earlier is intentional?

Reply:
ya I needed that to be done before installing other things.

Reply:
Hey @jessfraz! Is there any way of making your implementation work without changing the order here? #1598 parallelized the custom script a few months back and we were able to get to sub 5 minutes deployments but this PR adds back an extra minute which means we're now at > 6 min deployments for 1 master clusters... Thoughts?

Reply:
so the problem is this is installing things so I needed to make sure the other was no longer holding dpkg locks so this can install.... so maybe we can have an if runtime is clear-containers it takes forever and otherwise it will go back to the bottom and be fast?

I can make a PR if thats ok

Reply:
Yeah I think that would work. Taking the extra minute for clear containers is definitely better than slowing down deployment for every cluster since most users will be using the default I think :) Thanks!